accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -2458,7 +2460,14 @@ All other types are not always-exhaustive. Then:
     other option is to throw an error and most Dart users prefer to catch those
     kinds of mistakes at compile time.*
 
-[exhaustiveness]: https://github.com/dart-lang/language/blob/master/accepted/future-releases/0546-patterns/exhaustiveness.md
+*   It is a compile-time warning if a case in a switch statement, collection
```

Comment:
I (still) don't think we should specify warnings.

I'd prefer to tell the analyzer devs that they are free, and encouraged, to give a warning in this case, but don't make it part of the language specification.

This is just dead code. We have lots of dead code that we don't specify warnings for.

(Otherwise, phrasing is fine!)

Reply:
Good point, done.